builder/postaggregation/post_aggregator.go

```diff
@@ -62,6 +62,8 @@ func Load(data []byte) (builder.PostAggregator, error) {
 		p = NewLongLeast()
 	case "quantilesFromTDigestSketch":
 		p = NewQuantilesFromTDigestSketch()
+	case "quantilesDoublesSketchToQuantile":
```

Comment:
TDigest uses `from` v/s DoublesSketch is using `To`? can we use the same naming convention?

Reply:
it is based on the type specified here: https://druid.apache.org/docs/latest/development/extensions-core/datasketches-quantiles.html#quantile